guessNumber.js

```diff
@@ -0,0 +1,20 @@
+
+const guessNumber = function () {
+    
+    let randomNumber = Math.floor(Math.random()* 10), 
```

Comment:
`Math.random()` returns value between 0 and 1, including 0 but exclusive of 1, so we will never get 10 as a value for `randomNumber`. So current range of numbers is 0-9. Either change this `'Guess a number between 0 and 10:'` to `'Guess a number between 0 and 9:'` or play with creating 0-10 range.